### DISCUSSION TITLE
Allow adding as many custom shortcuts as desired

### DISCUSSION THREAD

**Discussion by avi12** [dailydotdev/daily#1387] [Feature Requests]
I have a list of sites that I visit daily, therefore I wish not to be limited by the number of custom shortcuts

**Comment by idoshamun** [comment]
Hi @avi12,

Thanks a lot for the suggestion. We'll consider it

**Comment by avi12** [comment]
Any plans to implement it?

**Comment by sabdailydev** [comment]
@avi12 Not immediately, but we will definitely pick it up when we get the engineering bandwidth to do this.
